common/test/acceptance/tests/lms/test_oauth2.py

```diff
@@ -44,13 +45,25 @@ def test_cancelling_redirects(self):
         assert self.oauth_page.visit()
         self.oauth_page.cancel()
 
+        def check_for_chrome():
+            query = self._qs(self.browser.title)
+            return 'access_denied' in query['error']
+
+        def check_for_firefox():
+            query = self._qs(self.browser.current_url)
```

Comment:
Why does `self.browser.title` not work in firefox?  That is a query that is used a lot inside of bok choy.